spec/index.bs

```diff
@@ -3502,11 +3502,16 @@ Issue: link to the semantics of SV_SampleIndex and SV_Coverage in WGSL spec.
                             - If the output SV_Coverage semantics is [=statically used=] by
                                 |descriptor|.{{GPURenderPipelineDescriptor/fragmentStage}}:
                                 - |descriptor|.{{GPURenderPipelineDescriptor/alphaToCoverageEnabled}} is `false`.
-                            - If |descriptor|.{{GPURenderPipelineDescriptor/primitiveTopology}} is
-                                {{GPUPrimitiveTopology/"line-strip"}} or
-                                {{GPUPrimitiveTopology/"triangle-strip"}}:
-                                - |descriptor|.{{GPURenderPipelineDescriptor/vertexState}}.{{GPUVertexStateDescriptor/indexFormat}}
-                                    is not `undefined`.
+                            - If |descriptor|.{{GPURenderPipelineDescriptor/primitiveTopology}} is:
+                                <dl class="switch">
```

Comment:
Aside: I noticed the `<dl class=switch>` styling doesn't need the extra yellow line I put on there for definitions. We should change the CSS rule for `dl` to `dl:not(.switch)` I think.

Reply:
Good call! Updated the style in the latest patch.